CONTRIBUTING.md

```diff
@@ -114,8 +114,9 @@ maximize the chances of your PR being merged.
 * If there is a question on who should review a PR please discuss in Slack.
 * Anyone is welcome to review any PR that they want, whether they are a maintainer or not.
 * Please **clean up the commit message** before merging. By default, GitHub fills the squash merge
-  commit message with every individual commit from the PR. Generally, we want a commit message
-  that is roughly equal to the original PR title and description.
+  commit message with every individual commit from the PR. Generally, the maintainer doing the merge
```

Comment:
since the commit is presented as two distinct fields in github can we refer to them as title and body or title and extended description? i think it would make this a little easier to understand rather than using message to refer to everything.

Reply:
`Please clean up the commit title and body before merging`

`fills the squash merge commit body`

`should overwrite the body with the original PR description (cleaning it up if necessary) while preserving the author's final DCO sign-off`

then i think an extra sentence about how titles should look.